internal/compiler/passes/embed_glyphs.rs

```diff
@@ -191,7 +191,14 @@ pub fn embed_glyphs<'a>(
     )
     .expect("internal error: fontdb returned a font that ttf-parser/fontdue could not parse");
                 embed_font(
-                    fontdb.face(face_id).unwrap().family.clone(),
+                    fontdb
+                        .face(face_id)
+                        .unwrap()
+                        .families
+                        .first()
+                        .expect("TrueType font without english family name encountered")
```

Comment:
I wonder if it would not be better to have compiler error (or even just warnigs)  instead of panicking in there.

Reply:
If it were a warning, then you'd get a panic on the MCU at run-time due to the lack of a font. I think that's worse, because there might not even be a well visualised panic handler in place. The only scenario I can imagine where this would happen is when a really buggy (almost corrupted) ttf is installed and picked up.

Reply:
Ok

Reply:
I'll submit this first and make a separate PR for the error handling.